src/__tests__/ActionMenu.test.tsx

```diff
@@ -137,7 +137,8 @@ describe('ActionMenu', () => {
         <MixedSelection />
       </ThemeProvider>,
     )
-    const button = component.getByLabelText('Group by')
+
+    const button = component.getAllByRole('button')[0]
```

Comment:
Is it easy to find the label text so we can keep using `getByLabelText` or not really? Would be awesome to keep `getByLabelText` here unless we are explicitly checking the ordering of the buttons. Hope that makes sense, let me know if I'm misunderstanding anything 👀

Reply:
I haven't found a way to get the Button by the label text via this testing suite which is why I went to this more generic approach. You are correct that it would be better to retrieve this in another way. I am just not sure how to yet. Do you have any ideas? I have tried:

- `component.getByLabelText('Group by Stage')`

Reply:
@JoshBowdenConcepts I bet this would work:

```suggestion
    const button = component.getByRole('button', {
      name: 'Group by Stage',
    })
```

Sadly I think `getByLabelText` only works with `<label>` so we need to either `getByText` or `getByRole` with the `name` matcher if we want to also check for the button semantics.

Reply:
This is really cool I have never used a specifier like that! I believe this works I will merge it in!